File: app/src/main/java/com/commit451/gitlab/model/api/Identity.java
package com.commit451.gitlab.model.api;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

@Parcel
public class Identity {
    @Json(name = "provider")
    String provider;
    @Json(name = "extern_uid")
    String externUid;

    public Identity() {}

    public String getProvider() {
        return provider;
    }

    public String getExternUid() {
        return externUid;
    }
}

File: app/src/main/java/com/commit451/gitlab/model/api/Identity.kt
package com.commit451.gitlab.model.api

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Identity(
    @Json(name = "provider")
    var provider: String? = null,
    @Json(name = "extern_uid")
    var externUid: String? = null
) : Parcelable
